src/Factory/AdminUserFactory.php

```diff
@@ -51,4 +51,13 @@ public static function setUsername(string $firstname, string $lastname): string
     {
         return $firstname . '_' . $lastname . '_' . random_int(1, 100);
     }
+
+    private static function getLocale(GoogleUser $googleUser): string
+    {
+        if (null === $googleUser->getLocale()) {
+            return 'fr_FR';
```

Comment:
Can you use current locale ?

```php
use Sylius\Component\Locale\Context\LocaleContextInterface;
...
public function __construct(private LocaleContextInterface $localeContext) {}

```
and 

```suggestion
            return $this->localeContext->getLocaleCode();
```